src/character.cpp

```diff
@@ -1011,7 +1011,8 @@ int Character::swim_speed() const
 
 bool Character::is_on_ground() const
 {
-    return get_working_leg_count() < 2 || has_effect( effect_downed );
```

Comment:
There are a lot other places where `effect_downed` is used. Should these places be updated as well?

Reply:
Discussed earlier. There is a huge variety of reasons `effect_downed` is used. I don't think this is in this PR scope to identify and change them